src/main/java/org/casbin/jcasbin/model/Model.java

```diff
@@ -77,6 +77,9 @@ public boolean addDef(String sec, String key, String value) {
         }
 
         model.get(sec).put(key, ast);
+
```

Comment:
Like I said, Model should not see an enforcer, nor it can reset the evaluator instance.

Reply:
I agree, that's why I first suggested the 'version' field https://github.com/casbin/jcasbin/issues/78#issuecomment-641334514

Anyway, I will come with a new proposition soon.